Package.swift

```diff
@@ -0,0 +1,68 @@
+// swift-tools-version:5.3
+
+import PackageDescription
+
+let package = Package(
+    name: "FluentUI",
+    defaultLocalization: "en",
+    platforms: [
+        .iOS(.v13),
+        .macOS(.v10_14),
+    ],
+    products: [
+        .library(
+            name: "FluentUI",
+            type: .static,
+            targets: [
+                "FluentUI"
+            ]
+        )
+    ],
+    targets: [
+        .target(
+            name: "FluentUI",
+            dependencies: [
+                .target(name: "FluentUI_ios", condition: .when(platforms: [.iOS])),
+                .target(name: "FluentUI_macos", condition: .when(platforms: [.macOS]))
+            ],
+            path: "public"
+        ),
+        .target(
+            name: "FluentUIResources",
+            path: "apple"
+        ),
+        .target(
+            name: "FluentUI_ios",
+            dependencies: [
+                .target(name: "FluentUIResources")
+            ],
+            path: "ios/FluentUI",
+            exclude: [
+                "Avatar/Avatar.resources.xcfilelist",
+                "BarButtonItems/BarButtonItems.resources.xcfilelist",
+                "Bottom Commanding/BottomCommanding.resources.xcfilelist",
+                "Core/Core.resources.xcfilelist",
+                "HUD/HUD.resources.xcfilelist",
+                "Info.plist",
+                "Navigation/Navigation.resources.xcfilelist",
+                "Notification/Notification.resources.xcfilelist",
+                "Other Cells/OtherCells.resources.xcfilelist",
+                "Resources/Localization/CultureMapping.json",
+                "Table View/TableView.resources.xcfilelist",
+                "Tooltip/Tooltip.resources.xcfilelist",
+                "Tooltip/Tooltip.resources.xcfilelist",
+                "TwoLineTitleView/TwoLineTitleView.resources.xcfilelist",
+            ]
```

Comment:
Why do we have to exclude these? (For my own knowledge's sake)

Reply:
Short answer: because Xcode gives a warning if you don't 🤠

Longer answer: AFAIK the Swift packager wants to understand every file that it sees in its paths, and it doesn't know what to do with these. They're not source and they're not resources, so it gives a warning saying "I don't know what this is". Putting those files in the exclusion list lets us skip over trying to parse them.

Remember: unlike an Xcode project, where files are opt-in (by manually adding them to the project), in SPM files are opt-out, and are considered part of the package by default unless you explicitly exclude them.

Reply:
Thanks for the explanation!